src/searchForm/autoComplete/overlay/AutoCompleteOverlay.tsx

```diff
@@ -14,24 +14,38 @@ export type AutoCompleteOverlayProps = Omit<
   readonly renderAutocompleteComponent: (
     props: Omit<AutoCompleteProps, 'searchForItems'>,
   ) => JSX.Element
+  readonly closeOnBlur: () => void
 }
 
 export const AutoCompleteOverlay = ({
   renderAutocompleteComponent,
+  closeOnBlur,
   ...props
 }: AutoCompleteOverlayProps) => {
+  const overlayRef = useRef(null)
+
   const icon = (
     <div className="kirk-autoComplete-icon">
       <Bullet type={BulletTypes.SEARCH} />
     </div>
   )
-
-  return renderAutocompleteComponent({
-    ...props,
-    autoFocus: true,
-    inputAddon: icon,
-    embeddedInSearchForm: true,
-  })
+  return (
+    <div
```

Comment:
No `className`?
`DatePickerOverlay` and `StepperOverlay` have one.

Reply:
No need an extra div here, the class is applied directly on the Autocomplete

Reply:
It's applied on `Autocomplete` component directly, but for consistency I'll move it here.